### PR TITLE
contribute.rst

### PR DESCRIPTION
**Pull request by MartaS-Easygov**
## Review

- [x] Ensure your files are written following RST specs (*not MD!*)
- [ ] Italian version
- [x] English version
- [ ] Example files 
- [x] Ask for review

### CODE REVIEW

File: docs/en/contribute.rst [hunk 1]
```diff
@@ -2,7 +2,15 @@
...
+Here are some tools that you can use to submit your contribution:
+
+- **Pull requests**. PRs let you tell others about changes you have pushed to a branch in a repository on GitHub. Once a pull request is opened, you can discuss and review the potential changes with collaborators and add follow-up commits before your changes are merged into the base branch;
+- **GitHub issues**. Issues let you track your work on GitHub, where development happens. When you mention an issue in another issue or pull request, the issue’s timeline reflects the cross-reference so that you can keep track of related work. To indicate that work is in progress, you can link an issue to a pull request. When the pull request merges, the linked issue automatically closes;
+- **Developers Italia Slack channel**. Slack is a messaging app for business that connects people to the information they need. *Developers Italia* is an open community based on the participation and contribution of public administrations, developers, technicians, students and citizens. *Developers Italia* started a Slack channel that everyone can join for free, where you can discover all their activities and participate in the discussion.
```
Comment (peppelinux) [inline docs/en/contribute.rst]: please add an invitation badge, as already in the README (but in markdown, please translate to RST accordingly)

````
[![Get invited](https://slack.developers.italia.it/badge.svg)](https://slack.developers.italia.it/)
````

**Review by peppelinux — CHANGES_REQUESTED** [review]

**Review by fmarino-ipzs — APPROVED** [review]

File: docs/en/contribute.rst [hunk 2]
```diff
@@ -2,7 +2,15 @@
...
 .. _contribute.rst:
 
-contribute.rst
+How to contribute
 +++++++++++++++++++++++++++
 
-Instruction to join in the development here.
+The IT Wallet project and, therefore, the present document are inspired by an **“open” spirit**. This means that the development is meant to be accessible by anyone and all interested parties are invited to join the process.
+
+Hence, stakeholders and members of the national and international community are encouraged, and more than welcome, to contribute to the content and editing of these technical rules.
```
Comment (peppelinux) [inline docs/en/contribute.rst]:
```suggestion
Hence, stakeholders and members of the national and international community are encouraged, and more than welcome, to contribute to the content of these technical rules.
```

File: docs/en/contribute.rst [hunk 3]
```diff
@@ -2,7 +2,15 @@
...
+
+Here are some tools that you can use to submit your contribution:
```
Comment (peppelinux) [inline docs/en/contribute.rst]:
```suggestion
Below are listed the ways that can be used to give contributions to this project:
```

File: docs/en/contribute.rst (hunk 1 above)
Comment (peppelinux) [inline docs/en/contribute.rst]:
```suggestion
- **GitHub issues**. By opening an issue it is possible to ask clarifications, propose enhancements or notify editorial typos. To indicate that work is in progress is possible to link an issue to a pull request;
- **Pull requests**. These represents active contributions to the project, where a discussion in the form of issues has already taken place. Once a pull request is opened, some discussions and reviews of the proposed changes take place, before the changes are merged into the main branch (`versione-corrente`);
- **Developers Italia Slack channel**. Slack is a messaging app for business that connects people to the information they need. *Developers Italia* is an open community based on the participation and contribution of public administrations, developers, technicians, students and citizens. *Developers Italia* started a Slack channel that [everyone can join for free](https://slack.developers.italia.it/), where you can discover all their activities and participate in the discussion.
```

**Review by peppelinux — CHANGES_REQUESTED** [review]

**Review by peppelinux — APPROVED** [review]

**Pull request merged**